docs/reference/logging/index.md

```diff
@@ -25,6 +25,16 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
+
+   --log-opt max-size=(unlimited)
+   --log-opt max-file=1
```

Comment:
yes. That is correct. `max-file` is meaningless without max-size. It won't be honored even if you set it. If you want me to describe it differently, Im good with that.